autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/SchedulePreview.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  ListItem, List, ListItemText,
+} from '@material-ui/core';
+import GenericCard from '../../GenericCard/GenericCard';
+import { RootState } from '../../../redux/reducer';
+import selectSchedule from '../../../redux/actions/selectedSchedule';
+import Meeting from '../../../types/Meeting';
+import Section from '../../../types/Section';
+
+const SchedulePreview: React.FC = () => {
+  const schedules = useSelector<RootState, Meeting[][]>((state) => state.schedules);
+  const selectedSchedule = useSelector<RootState, number>((state) => state.selectedSchedule);
+  const dispatch = useDispatch();
+
+  const renderSchedule = (schedule: Meeting[], idx: number): JSX.Element => (
+    <ListItem
+      button
+      alignItems="flex-start"
+      key={idx}
+      onClick={(): void => { dispatch(selectSchedule(idx)); }}
+      selected={selectedSchedule === idx}
+    >
+      <ListItemText
+        primary={(
+          <span>
+            <span>{`Schedule ${idx + 1}`}</span>
+            <span style={{ float: 'right' }}>3.65 GPA</span>
+          </span>
+        )}
+        secondary={
+          // get unique sections, assuming that meetings from the same section are adjacent
+          schedule.reduce((acc, curr): Section[] => {
+            const lastSection = acc[acc.length - 1];
+            if (!lastSection || lastSection.id !== curr.section.id) {
+              return acc.concat(curr.section);
+            }
+            return acc;
+          }, []).map((sec: Section) => (
+            <span key={sec.id}>
+              {`${sec.subject} ${sec.courseNum}-${sec.sectionNum}`}
+              <br />
+            </span>
+          ))
+        }
+      />
+    </ListItem>
+  );
+
+  return (
+    <GenericCard
+      style={{ flexGrow: 1, flexBasis: 0, overflowY: 'auto' }}
```

Comment:
I'd rather we make a `SchedulePreview.css` for the styles here, if there were only one or two then inline would be fine imo but since there are 5 I think making a new file for them is worthwhile. Wondering what @gannonprudhomme thinks about this

Same applies for `OptionsCard`, there's already a .css file for it but still 3 inline styles

Reply:
Yeah I agree, not sure about `OptionsCard` but we should def do it for `SchedulePreview`